internal/lpastore/update.go

```diff
@@ -143,8 +147,16 @@ func (c *Client) SendAttorney(ctx context.Context, donor *Lpa, attorney *actor.A
 			)
 		}
 	} else {
+		if attorney.Email != "" {
+			body.Changes = append(body.Changes, updateRequestChange{Key: attorneyKey + "/email", New: attorney.Email, Old: lpaAttorney.Email})
+		}
+
+		if lpaAttorney.Channel == actor.ChannelPaper {
```

Comment:
will you do trust corporations separately? (I keep forgetting about them)

Reply:
I haven't checked the backlog for a ticket but yeah I'd been treating them differently and assuming it would be its own piece of work.